README.md

```diff
@@ -78,7 +83,13 @@ The steering committee delegates ownership of various Kubernetes community accou
 | sc2@kubernetes.io | Paris Pittman |
 | sc3@kubernetes.io | Aaron Crickenberger |
 
-## Contact
+## Zoom Admins
 
-If you need the steering committee please email the list steering@kubernetes.io ([archive](https://groups.google.com/a/kubernetes.io/forum/#!forum/steering))
-or reach out in [#steering-committee](https://kubernetes.slack.com/messages/steering-committee) on Slack.
+The following committee members have zoom credentials for the `steering-private@kubernetes.io` account.
+However, all Steering Committee members have access to the zoom host key.
```

Comment:
@lachie83 fyi you'll find this as a pinned message in the `steering-private` slack channel

Reply:
also added this in the onboarding doc